src/STAT_lmonFrontEnd.C

```diff
@@ -0,0 +1,742 @@
+#include "STAT_lmonFrontEnd.h"
+
+using namespace std;
+using namespace MRN;
+
+//! the LMON status bit vector to detect daemon and application exit
+static int gsLmonState = 0;
+
+// start timer for composite operations
+extern STAT_timer gTotalgStartTime;
+
+// start timer for individual operations
+extern STAT_timer gStartTime;
+
+// end timer for individual operations
+extern STAT_timer gEndTime;
+
+
+STAT_lmonFrontEnd::STAT_lmonFrontEnd()
```

Comment:
The code in this file was almost entirely cut and pasted from STAT_FrontEnd.C.   I've commented on them in that file.